uber/templates/preregistration/badge_choice.html

```diff
@@ -100,10 +100,20 @@ <h2 style="margin-top:10px"> Choose a Badge </h2>
     {% endif %}
     </tr>
 {% endif %}
-{% if DISPLAY_ONEDAY_BADGES %}
+{% if DISPLAY_ONEDAY_BADGES or AT_THE_CON %}
     <tr>
         <td> <b>Single Day Passes:</b> </td>
-        <td><ul><li>Single day passes available for sale at the door for {% single_day_prices %}.</li></ul></td>
+        <td>
+            <ul>
+                <li>Single day passes available for sale at the door for {% single_day_prices %}.</li>
```

Comment:
Instead of modifying the main prereg form, which would be a lot of work, I just linked to the kiosk form.  That should hold us over for now.